packages/init/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slicemachine/init",
-  "version": "1.1.14-dev.0",
+  "version": "1.1.14-dev.9",
```

Comment:
you didn't change the versions of the core and Clients in the dependencies to match the new versions.